addons/addon-base-raas/packages/base-raas-post-deployment/lib/steps/create-service-catalog-portfolio.js

```diff
@@ -295,13 +300,55 @@ class CreateServiceCatalogPortfolio extends Service {
         DisableTemplateValidation: true,
         Type: 'CLOUD_FORMATION_TEMPLATE',
         Description: productToCreate.description || autoCreateDesc,
+        Name: nextProvArtifact,
       },
       ProductId: productId,
     };
     const data = await servicecatalog.createProvisioningArtifact(params).promise();
     return data.ProvisioningArtifactDetail.Id;
   }
 
+  async getNextArtifactVersion(latestVersionInSC) {
+    // For product artifacts in Service Catalog, versions and names are the same thing
+    // If no previous version found, assign 'v2' (this will help in next update cycle)
+    let returnVal = 'v2';
+    // Only finds version strings that match patterns:
+    // "V<n>", "V<n>.0", "V<n>.0.0", "v<n>", "v<n>.0", "v<n>.0.0"
+    const pattern = /^(v|V)(\d+\.)?(\d+\.)?(\*|\d+)$/;
+    let latestVersionNo;
+    if (latestVersionInSC) {
+      const parsedOutput = latestVersionInSC.match(pattern);
+      this.log.info(`${parsedOutput}`);
```

Comment:
Do we want to log this? doesn't seem to have context around it

Reply:
For testing only, needs to be taken out. Thanks for pointing it out.